src/lib/fees/model.ts

```diff
@@ -48,15 +53,26 @@ export const calculateFees: CalculateFeeBps = ({ tradeAmountUsd, foxHeld, feeMod
   const minFeeBps = bn(FEE_CURVE_MIN_FEE_BPS)
   const midpointUsd = bn(FEE_CURVE_MIDPOINT_USD)
   const feeCurveSteepness = bn(FEE_CURVE_STEEPNESS_K)
+  const isThorFreeEnabled = getConfig().REACT_APP_FEATURE_THOR_FREE_FEES
 
   // trades below the fee threshold are free.
-  const isFree = tradeAmountUsd.lt(noFeeThresholdUsd)
+  const isFree =
+    tradeAmountUsd.lt(noFeeThresholdUsd) ||
+    (isThorFreeEnabled && thorHeld?.isGreaterThanOrEqualTo(THORSWAP_UNIT_THRESHOLD))
```

Comment:
tracking this in `isThorFree`

Reply:
Done in [05daabc](https://github.com/shapeshift/web/pull/8044/commits/05daabc89d2082b788ce937e871349e89d2193ba)